web/src/beta/features/PluginPlayground/Plugins/presets/layers/add-geojson.ts

```diff
@@ -0,0 +1,109 @@
+import { FileType, PluginType } from "../../constants";
+
+const yamlFile: FileType = {
+  id: "layers-add-geojson-reearth-yml", 
```

Comment:
```suggestion
  id: "example-plugin-layers-add-geojson", 
```
This should be the id of this plugin. Naming it as xx-reearth-yml seems not reasonable.